src/Text/Pandoc/Writers/Docx/OpenXML.hs

```diff
@@ -171,6 +171,58 @@ makeTOC opts = do
       ])
     ]] -- w:sdt
 
+makeLOF :: (PandocMonad m) => WriterOptions -> WS m [Element]
+makeLOF opts = do
+  let lofCmd = "TOC \\h \\z \\t \"Image Caption\" \\c" :: Text
+  lofTitle <- gets stLofTitle
+  -- Create a separate style for LOF Heading?
```

Comment:
Can we use the same style that is used for the TOC heading?
Otherwise, maybe a separate style makes sense.

Reply:
We can use the same style.